code/game/machinery/pipe/pipe_dispenser.dm

```diff
@@ -183,6 +185,7 @@ Nah
 	icon = 'icons/obj/stationobjs.dmi'
 	icon_state = "pipe_d"
 	density = TRUE
+	desc = "Dispenses pipes that will move humans and/or trash around."

```

Comment:
transit tubes aren't meant for trash

Reply:
I put down "and/or" to make a subtle joke that the people are sometimes the trash. Bad joke I guess